z/mremap_linux_arm64.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Should we have a build directive like `//  +build arm64` here?

Reply:
Not needed, the `_arm64` suffix is recognized by the go compiler

Reply:
I see, good to know